components/library/libmng/manifests/sample-manifest.p5m

```diff
@@ -10,7 +10,7 @@
 #
 
 #
-# Copyright 2016 <contributor>
+# Copyright 2018 <contributor>
```

Comment:
This is not required unless anything actually changed. `git checkout -- manifests/sample-manifest.p5m` will do.